Add 2FA checkup guide (#830)
lukso-network/docs: docs/faq/onboarding/extension.md

:::

## Can I add 2FA recovery to an existing Universal Profile?

> At the current state (11th January 2024), the 2FA recovery method from LUKSO can only be added during the creation of a profile on [UniversalProfile.cloud](https://universalprofile.cloud/).

Reviewer: ```suggestion
At the current state (11th January 2024), the 2FA recovery method from LUKSO can only be added during the creation of a Universal Profile on [UniversalProfile.cloud](https://universalprofile.cloud/).
```

## How to check if 2FA recovery is set up correctly?

> If you ever want to recover your profile, a fresh controller key will be added to this profile from a new extension setup. You must verify multiple steps to check if this new controller key can be added to your Universal Profile by the recovery service:

Reviewer: ```suggestion
If you ever want to recover your Universal Profile, a fresh controller key will be added to this profile from a new extension setup. You must verify multiple steps to check if this new controller key can be added to your Universal Profile by the recovery service:
```

> If you ever want to recover your profile, a fresh controller key will be added to this profile from a new extension setup. You must verify multiple steps to check if this new controller key can be added to your Universal Profile by the recovery service:

Reviewer: Not sure about using the word "key" alongside the word "controller". Would remove it for consistency with next paragraph

```suggestion
If you ever want to recover your profile, a new controller will be added to this profile from a new extension setup. You must verify multiple steps to check if this new controller can be added to your Universal Profile by the recovery service:
```

### 1. Verify the recovery controller in your extension

If you visit the controller menu in the Universal Profile Browser Extension, there need to be at least three controllers:

- The Universal Profile Browser Extension (used for profile interactions)
- The Universal Receiver (used to write asset updates on the profile)
- The Universal Profile Recovery (used as a backup for the profile)

To confirm that the permissions of the recovery controller were adequately set up, click on the controller of the Universal Profile Recovery and verify that the **Add Controller** permission is enabled.

![2FA Controller](../../../static/img/faq/2fa-controller.png)

:::info Controller Names

The names of the controllers can vary, depending on the time of your Universal Profile setup.

:::

### 2. Verify the recovery controller on the profile explorer

> Next, we must check if the recovery was successfully added via the LUKSO Relay Service. **Copy the address of your Universal Profile Recovery** from the controller menu of the Universal Profile Extension and open [UniversalProfile.cloud](https://universalprofile.cloud/?network=mainnet). On the profile explorer, **input the address** of the Universal Profile Recovery **into the search bar**. If it was successfully added, it will show a recovery profile.

Reviewer: Suggest doing a numbered list here:

```suggestion
Next, we must check if the recovery was successfully added via the LUKSO Relay Service. 

1. **Copy the address of your Universal Profile Recovery** from the controller menu of the Universal Profile Extension
2. open [UniversalProfile.cloud](https://universalprofile.cloud/?network=mainnet). 
3. On the profile explorer, **input the address** of the Universal Profile Recovery **into the search bar**. 

If it was successfully added, it will show a recovery profile.
```

Reviewer: I am not sure to understand. How does finding the recovery UP in the universalprofile.cloud explorer gives you the guarantee that this recovery UP has been setup for your UP?

It should rather be something to check on the dashboard of my.universalprofile.cloud no?

Author: It is not registered on the relayer if it has not been added to the profile. At least that's what I was told.

> ![Recovery Search](../../../static/img/faq/recovery-search.png)

Reviewer: Great to add images and screenshots here 👌🏻
